crawler/file_processing.py

```diff
@@ -132,20 +132,18 @@ def get_files_in_download_dir(self) -> List[str]:
             return []
 
     def clean_up(self) -> None:
-        """Remove the files downloaded from the SFTP for the given centre.
-
-        Arguments:
-            centre {Dict[str, str]} -- the centre in question
-        """
+        """Remove the files downloaded from the SFTP for the given centre."""
         logger.debug("Remove files")
         try:
             shutil.rmtree(self.get_download_dir())
+            (_, subfolders, files) = next(os.walk("tmp/files/"))
```

Comment:
What does this do? Find the next temporary file? Why is it needed now and not before?

Reply:
This snuck in while debugging...